readme.md

```diff
@@ -59,8 +59,9 @@ Type: `Object`
 
 ##### borderColor
 
-Type: `string`<br>
+Type: `string` or `function`<br>
 Values: `black` `red` `green` `yellow` `blue` `magenta` `cyan` `white` `gray`
+Value in the case of a function should be a `chalk.hex` or `chalk.rgb` or any other [chalk color models](https://github.com/chalk/chalk#256-and-truecolor-color-support)
```

Comment:
I don't think we should expose everything like this. We can just allow a string with a color HEX.